### PR TITLE
KAA-700: Administration guide

### PR DESCRIPTION
**Pull request by sashadidukh**

### CODE REVIEW

File: doc/Administration-guide/index.md [hunk 1]
```diff
@@ -5,4 +5,21 @@ permalink: /:path/
 sort_idx: 30
 ---
 
-Guide for the platform administrators.
\ No newline at end of file
+{% include variables.md %}
+
+The intention of this guide is to introduce Kaa platform administrative tasks for you.
```
Comment (ashvayka) [inline doc/Administration-guide/index.md]: The intention of this guide is to cover Kaa platform administrative tasks.

File: doc/Administration-guide/index.md [hunk 2]
```diff
@@ -5,4 +5,21 @@ permalink: /:path/
...
+
+# Sections overview
+
+This guide provide the overview of [key system components]({{root_url}}Administration-guide/System-components-overview/) from the administration point of
+view. Helps with [planning your deployment]({{root_url}}Administration-guide/System-installation/Planning-your-deployment/), provides an overview of
+[Kaa platform installation]({{root_url}}Administration-guide/System-installation/) on a
+[single Linux node]({{root_url}}Administration-guide/System-installation/Single-node-installation/) and administration of
```
Comment (ashvayka) [inline doc/Administration-guide/index.md]: [single Linux node]({{root_url}}Administration-guide/System-installation/Single-node-installation/) or

File: doc/Administration-guide/index.md [hunk 3]
```diff
@@ -5,4 +5,21 @@ permalink: /:path/
...
+[Kaa cluster on Linux nodes]({{root_url}}Administration-guide/System-installation/Cluster-setup/). After installation
```
Comment (ashvayka) [inline doc/Administration-guide/index.md]: [cluster of Linux nodes]({{root_url}}Administration-guide/System-installation/Cluster-setup/).

File: doc/Administration-guide/index.md [hunk 4]
```diff
@@ -5,4 +5,21 @@ permalink: /:path/
...
+[system configuration]() guide helps to tune Kaa depends on your requirements.
```
Comment (ashvayka) [inline doc/Administration-guide/index.md]: +[system configuration]() guide helps to tune Kaa based on your requirements.

**Comment by ashvayka** [comment]
Few minor comments, otherwise, LGTM

**Comment by sashadidukh** [comment]
Fix comments

**Pull request merged**